airbyte-db/lib/src/main/java/io/airbyte/db/instance/development/MigrationDevCenter.java

```diff
@@ -62,14 +62,19 @@ protected MigrationDevCenter(String schemaDumpFile) {
     return container;
   }
 
+  /**
+   * Directory under which a new migration will be created. This should match the database's name and match the {@link Db} enum.
+   */
+  protected abstract String getMigrationDirectory();
```

Comment:
Since this method is not needed anywhere else, can the directory be an instance variable instead of being returned from this abstract method?